src/main/java/org/sunbird/profile/service/ProfileServiceImpl.java

```diff
@@ -2019,6 +2026,27 @@ public String validateToken(String token) throws Exception {
 			throw new Exception("IOException has occurred: " , e);
 		}
     }
+
+	/**
+	 * Method to validate if the provided email or phone number already exists in the system.
+	 *
+	 * @param profileDetailsMap A map containing the profile details.
+	 * @return Constants.OK if neither email nor phone number exists, Constants.EMAIL_EXIST_ERROR if email already exists,
+	 *         Constants.PHONE_NUMBER_EXIST_ERROR if phone number already exists.
+	 */
+	private String validateExistingPhoneEmail(Map<String, Object> profileDetailsMap) {
+		// Extract personal details from the profileDetailsMap
+		Map<String, Object> personalDetails = (Map<String, Object>) profileDetailsMap.get(Constants.PERSONAL_DETAILS);
+		// Check if the provided email already exists in the system
+		if (userUtilityService.isUserExist(Constants.EMAIL, (String) personalDetails.get(Constants.EMAIL))) {
```

Comment:
There is a possibility that profileDetails object doesn't have personalDetails... 
Do check personalDetails exist -- also check personalDetails contains key -> Email -- if so then call isUserExist method